Dockerfile

```diff
@@ -76,49 +76,51 @@ SHELL ["/bin/bash", "--login", "-c"]
 
 
 # Build the final image with Prefect installed and our entrypoint configured
-FROM ${BASE_IMAGE} as final
+FROM ${BASE_IMAGE} AS final
 
-ENV LC_ALL C.UTF-8
-ENV LANG C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
 
-LABEL maintainer="help@prefect.io"
-LABEL io.prefect.python-version=${PYTHON_VERSION}
-LABEL org.label-schema.schema-version = "1.0"
-LABEL org.label-schema.name="prefect"
-LABEL org.label-schema.url="https://www.prefect.io/"
+LABEL maintainer="help@prefect.io" \
+    io.prefect.python-version=${PYTHON_VERSION} \
+    org.label-schema.schema-version="1.0" \
+    org.label-schema.name="prefect" \
+    org.label-schema.url="https://www.prefect.io/"
 
 WORKDIR /opt/prefect
 
-# Install requirements
-# - tini: Used in the entrypoint
-# - build-essential: Required for Python dependencies without wheels
-# - git: Required for retrieving workflows from git sources
+# Install system requirements
 RUN apt-get update && \
     apt-get install --no-install-recommends -y \
     tini=0.19.* \
     build-essential \
     git=1:2.* \
+    curl \
+    ca-certificates \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
-# Pin the pip version
-RUN python -m pip install --no-cache-dir pip==24.2
+# Install UV
+ADD https://astral.sh/uv/install.sh /uv-installer.sh
+RUN sh /uv-installer.sh && rm /uv-installer.sh
+ENV PATH="/root/.local/bin:$PATH"
```

Comment:
Why not copy uv from the official docker image?

```
COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
```

As [recomended by uv](https://docs.astral.sh/uv/guides/integration/docker/#installing-uv)

Reply:
good point! i went back to the `COPY --from` in [b815117](https://github.com/PrefectHQ/prefect/pull/16235/commits/b815117e7a62e2a05cfc0f3fe7bc2d7976e6c1b8)

i was curious to see the differences in build time between the options